wgs/wasm/deliverables/wasm-oci-artifact.md

```diff
@@ -0,0 +1,132 @@
+---
```

Comment:
FYI. After https://github.com/cncf/tag-runtime/pull/137 is merged, we need to put this file under the folder `wgs/wasm/deliverables/wasm-oci-artifact/` and rename it to `index.md`

Reply:
I've updated the path in preperation